src/templates/React InstantSearch Hooks/src/App.tsx.hbs

```diff
@@ -0,0 +1,99 @@
+import algoliasearch from 'algoliasearch/lite';
+import {
+  {{#if flags.dynamicWidgets}}
```

Comment:
these imports can be like https://github.com/algolia/create-instantsearch-app/blob/d9358cde3b212fe29d5f35d47c3a0de7250d9d50/src/templates/React%20InstantSearch/src/App.js.hbs#L7-L19 with fallbackWidget